styles.css

```diff
@@ -3,12 +3,33 @@
     --menu-bar-height: 10px;
 }
 
+.container {
+    padding: 20px 0 20px 0;
```

Comment:
In de uitwerking wordt ook het volgende toegepast, verder geen opmerkingen:
`display: flex;`
   ` justify-content: center;`
   ` align-items: center;`